packages/client/src/runtime/utils/getRuntime.ts

```diff
@@ -1,4 +1,68 @@
-import { detectRuntime, Runtime } from 'detect-runtime'
```

Comment:
Had we replaced this with `std-env` altogether`, we'd have a 3KB larger `@prisma/client` and adapters bundle, see https://github.com/prisma/prisma/pull/26420.